tests/middleware/test_message_state.py

```diff
@@ -6,25 +6,39 @@
 from remoulade.cancel import Cancel
 from remoulade.middleware import Middleware, SkipMessage
 from remoulade.state.backend import State, StateNamesEnum
-from remoulade.state.middleware import MessageState
+from remoulade.state.middleware import MessageState, State
 
 
 class TestMessageState:
     """Class to test the middleware
     MessageState
     """
 
-    def test_pending_state_message(self, stub_broker, state_middleware, do_work):
+    def test_pending_state_message(self, stub_broker, state_middleware, do_work, frozen_datetime):
         msg = do_work.send()
-        assert state_middleware.backend.get_state(msg.message_id).name == StateNamesEnum.Pending
+        state = state_middleware.backend.get_state(msg.message_id)
+        assert state.message_id == msg.message_id
+        assert state.name == StateNamesEnum.Pending
+        assert state.enqueued_datetime.isoformat() == "2020-02-03T00:00:00+00:00"
 
-    def test_success_state_message(self, stub_broker, stub_worker, state_middleware, do_work):
+    def test_success_state_message(self, stub_broker, stub_worker, state_middleware, frozen_datetime):
+        @remoulade.actor
+        def do_work():
+            frozen_datetime.tick(delta=15)
+
+        stub_broker.declare_actor(do_work)
+        stub_worker.pause()
         msg = do_work.send()
+        stub_worker.resume()
```

Comment:
you are missing a  frozen_datetime.tick(delta=15) between do_work.send and worker.resume. We want 3 different datetimes a the end.